src/assets/javascripts/angular/services/analyticsService.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
+import {
+  googleAnalytics
+} from 'functions/googleAnalytics'
+
```

Comment:
I decided to share functionality with this new ES6 class, which references `window.ga`. `window.ga` is initialized/injected into the front-end environment by the analyticsService.js. An [event listener](https://github.com/sis-berkeley-edu/calcentral/blob/master/src/assets/javascripts/angular/services/analyticsService.js#L72,L73) is also still configured to trigger Google Analytics events for each page change via ngRouter.